src/binder/inc/bindertracing.h

```diff
@@ -49,6 +50,61 @@ namespace BinderTracing
         ReleaseHolder<PEAssembly> m_resultAssembly;
         bool m_cached;
     };
+
+
+    class ResolutionAttemptedOperation
+    {
+    private:
+        enum class Result : uint16_t
+        {
+            Attempt,
+            Success,
+            AssemblyNotFound,
+            MismatchedAssemblyName,
+            IncompatibleVersion,
+            Unknown,
+        };
+
+    public:
+        enum class Stage : uint16_t
+        {
+            NotYetStarted,
+            FindInLoadContext,
+            ALCLoad,
+            PlatformAssemblies,
+            DefaultALCFallback,
+            AppDomainAssemblyResolveEvent,
+        };
```

Comment:
These enum values should match the corresponding value maps in `ClrEtwAll.man`

Reply:
The `AssemblyLoadContextResolvingEvent` stage is also missing from the enum.

Reply:
Good catch.  Done.